sawmill-core/src/main/java/io/logz/sawmill/ConditionalFactoriesLoader.java

```diff
@@ -17,35 +19,25 @@
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
 public class ConditionalFactoriesLoader {
+
     private static final Logger logger = LoggerFactory.getLogger(ConditionalFactoriesLoader.class);
-    private static ConditionalFactoriesLoader instance;
+
     private final Reflections reflections;
     private final Map<Class<?>, Object> services;
```

Comment:
`services` is not a clear name. What do you think about renaming?
```suggestion
    private final Map<Class<?>, Object> dependenciesToInject;
```